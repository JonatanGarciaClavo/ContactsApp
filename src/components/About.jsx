import React from 'react';
import { Card, CardText, CardTitle } from 'material-ui';

const About = () => (
  <Card style={{ margin: '1em 0 0 0' }}>
    <CardTitle title="Contacts App" subtitle="Learnig redux" />
    <CardText>This is a example app using redux.</CardText>
  </Card>
);

export default About;
