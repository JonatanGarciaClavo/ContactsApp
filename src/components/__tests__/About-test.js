import React from 'react';
import About from '../About';

describe('Testing About component', () => {
  it('renders About using Snapshots', () => {
  	const wrapper = shallow(<About />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
