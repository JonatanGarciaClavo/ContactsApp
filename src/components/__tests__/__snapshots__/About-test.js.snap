exports[`Testing About component renders About using Snapshots 1`] = `
<Card
  expandable={false}
  expanded={null}
  initiallyExpanded={false}
  style={
    Object {
      "margin": "1em 0 0 0",
    }
  }>
  <CardTitle
    subtitle="Learnig redux"
    title="Contacts App" />
  <CardText>
    This is a example app using redux.
  </CardText>
</Card>
`;
